build/pause/Makefile

```diff
@@ -94,6 +95,11 @@ push: .push-$(ARCH)
 	docker push $(IMAGE_WITH_ARCH):$(TAG)
 	touch $@
 
+push-manifest:
+	docker manifest create --amend $(IMAGE):$(TAG) $(foreach ALL_ARCH,$(ALL_ARCH),$(IMAGE)-$(ALL_ARCH):$(TAG))
+	docker manifest annotate --arch $(ARCH) $(IMAGE):$(TAG) $(IMAGE_WITH_ARCH):$(TAG)
```

Comment:
Just one arch annotate is not good enough, you need to loop through all possible archs annotation. Refer code here:

https://github.com/kubernetes/kubernetes/blob/master/test/images/image-util.sh#L135:L137